python/paddle/text/datasets/imikolov.py

```diff
@@ -180,8 +195,8 @@ def _load_anno(self):
                 else:
                     raise AssertionError('Unknow data type')
 
-    def __getitem__(self, idx):
+    def __getitem__(self, idx: int) -> tuple[npt.NDArray[np.int_]]:
```

Comment:
```suggestion
    def __getitem__(self, idx: int) -> tuple[npt.NDArray[np.int_], npt.NDArray[np.int_]:
```

返回的是 `(src_seq, trg_seq)` ?

Reply:
恩，对，看漏了一个。